pages/1.11/installing/evaluation/cloud-installation/packet/index.md

```diff
@@ -1,30 +1,31 @@
 ---
 layout: layout.pug
-excerpt:
+excerpt: Running DC/OS on Packet
 title: Running DC/OS on Packet
 navigationTitle: Packet
-menuWeight: 3
+menuWeight: 25
+oss: true
 ---
 
 You can create a DC/OS cluster on Packet bare metal using Terraform.
```

Comment:
Should it be moved to on-prem installing guides?

Reply:
@urbanserj this will revised in Phase - 2.